modules/nw-egressnetworkpolicy-about.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * networking/openshift-sdn/configuring-egress-firewall.adoc
+
+[id="nw-egressnetworkpolicy-about_{context}"]
+= How an egress firewall works in a project
+
+As an {product-title} cluster administrator, you can use an _egress firewall_ to
+limit the external hosts that some or all Pods can access from within the
+cluster. Egress firewall policy supports the following scenarios:
+
+- A Pod can only connect to internal hosts, and cannot initiate connections to
+the public Internet.
+- A Pod can only connect to the public Internet, and cannot initiate connections
+to internal hosts that are outside the {product-title} cluster.
+- A Pod cannot reach specified internal subnets or hosts outside the {product-title} cluster that should be unreachable.
+- A Pod can connect to only specific external hosts.
+
+An egress firewall policy is configured by creating an EgressNetworkPolicy Custom Resource (CR) object and specifying an IP address range in CIDR format or by specifying a DNS name.
+For example, you can allow one project access to a specified IP range but deny the same access to a different project, or you can restrict application developers from updating from Python pip mirrors, and force updates to come only from approved sources.
+
+[IMPORTANT]
+====
+You must have OpenShift SDN configured to use either the network policy or multitenant modes to configure egress firewall policy.
+
+If you are using network policy mode, egress policy is compatible with only one policy per namespace, and will not work with projects that share a network, such as global projects.
+====
+
+[CAUTION]
+====
+Egress network policy rules do not apply to service endpoint traffic. An Ingress Controller handles traffic for a service, bypassing the egress firewall.
+====
+
+[id="limitations-of-an-egress-firewall"]
+== Limitations of an egress firewall
+
+An egress firewall has the following limitations:
+
+* No project can have more than one EgressNetworkPolicy object.
+
+* The `default` project cannot use egress network policy.
+
+* When using the OpenShift SDN network provider in multitenant mode the following limitations apply:
+
+  - Any project that has been made global via `oc adm pod-network make-projects-global` cannot use egress network policy.
+
+  - Projects merged together via `oc adm pod-network join-projects` cannot use egress network policy in any of the joined projects.
```

Comment:
s/together via  `oc adm pod-network join-projects`/ by using the  `oc adm pod-network join-projects` command